### PR TITLE
Add an install script

### PR DESCRIPTION
**Pull request by micziz**
# Description

This Pull Request adds 2 things.

Adds an install script. This is a continuation of #465.

# Issue Fixes

Resolves this comment in #419 by @Drugsosos "I can write install.sh for linux with all dependences and stuff"

# Checklist:

- [x] I am pushing changes to the **develop** branch
- [x] I am using the recommended development environment
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have formatted and linted my code using python-black and pylint
- [x] I have cleaned up unnecessary files
- [x] My changes generate no new warnings
- [x] My changes follow the existing code-style
- [x] My changes are relevant to the project

# Any other information (e.g how to test the changes)

None

### CODE REVIEW

**Comment by JasonLovesDoggo** [comment]
Sox should not be in the requirements.txt file

**Review by JasonLovesDoggo — CHANGES_REQUESTED** [review]
^

**Comment by micziz** [comment]
The install instructions does not mention sox. Is It needed?

**Comment by JasonLovesDoggo** [comment]
After much work me and Callum have managed to bypass it (Callum discovered athat we can use moviepy and implement it manually)

**Comment by micziz** [comment]
Ok, will remove

File: README.md [hunk 1]
```diff
@@ -42,6 +42,14 @@ The only original thing being done is the editing and gathering of all materials
 
 3. Run `playwright install` and `playwright install-deps`. (if this fails try adding python -m to the front of the command)
 
+**EXPERIMENTAL!!!!**
+
+On MacOS and Linux (debian, arch, fedora and centos, and based on those), you can run an install script that will automatically install steps 1 to 3. (requires bash)
+
+`bash <(curl -sL https://raw.githubusercontent.com/micziz/RedditVideoMakerBot/master/install.sh)`
```
Comment (CallumIO) [inline README.md:49]: This should be pointed at this repo

File: install.sh [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+
+# install playwright function
+function install_playwright(){
+    # tell the user that the script is going to install playwright 
+    echo "Installing playwright"
+    # cd into the directory where the script is downloaded
+    cd RedditVideoMakerBot
+    # run the install script
+    playwright install 
+    playwright install-deps
```
Comment (CallumIO) [inline install.sh:118]: These should be ran as `python -m` by default, it's highly unlikely the user will have playwright installed on their system

File: README.md (hunk 1 above)
Comment (micziz) [reply README.md]: Right!

File: install.sh (hunk 2 above)
Comment (micziz) [reply install.sh:118]: Playwright is installed with requirements.txt, but I will make the change

Comment (CallumIO) [reply install.sh]: Yh I meant by `pacman` or `apt`, all good

**Review by CallumIO — APPROVED** [review]
LGTM, thank you!

**Pull request merged**